src/main/java/io/odpf/firehose/metrics/Instrumentation.java

```diff
@@ -1,30 +1,17 @@
 package io.odpf.firehose.metrics;
 
-import io.odpf.firehose.message.Message;
 import io.odpf.firehose.error.ErrorType;
+import io.odpf.firehose.message.Message;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
 import java.time.Instant;
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Stream;
 
-import static io.odpf.firehose.metrics.Metrics.ERROR_EVENT;
-import static io.odpf.firehose.metrics.Metrics.ERROR_MESSAGES_TOTAL;
-import static io.odpf.firehose.metrics.Metrics.ERROR_MESSAGE_CLASS_TAG;
-import static io.odpf.firehose.metrics.Metrics.ERROR_TYPE_TAG;
-import static io.odpf.firehose.metrics.Metrics.FATAL_ERROR;
-import static io.odpf.firehose.metrics.Metrics.GLOBAL_MESSAGES_TOTAL;
-import static io.odpf.firehose.metrics.Metrics.MESSAGE_SCOPE_TAG;
-import static io.odpf.firehose.metrics.Metrics.MESSAGE_TYPE_TAG;
-import static io.odpf.firehose.metrics.Metrics.MessageType;
-import static io.odpf.firehose.metrics.Metrics.NON_FATAL_ERROR;
-import static io.odpf.firehose.metrics.Metrics.PIPELINE_END_LATENCY_MILLISECONDS;
-import static io.odpf.firehose.metrics.Metrics.PIPELINE_EXECUTION_LIFETIME_MILLISECONDS;
-import static io.odpf.firehose.metrics.Metrics.SINK_PUSH_BATCH_SIZE_TOTAL;
-import static io.odpf.firehose.metrics.Metrics.SINK_RESPONSE_TIME_MILLISECONDS;
-import static io.odpf.firehose.metrics.Metrics.SOURCE_KAFKA_MESSAGES_FILTER_TOTAL;
-import static io.odpf.firehose.metrics.Metrics.SOURCE_KAFKA_PULL_BATCH_SIZE_TOTAL;
+import static io.odpf.firehose.metrics.Metrics.*;
```

Comment:
don't put asterisk. It will fail the checkstyle.

Reply:
changes added